ietf/sync/rfceditor.py

```diff
@@ -467,35 +530,24 @@ def update_docs_from_rfc_index(index_data, errata_data, skip_older_than_date=Non
             e.by = system
             e.desc = "RFC published"
             e.save()
-            events.append(e)
+            rfc_events.append(e)
 
-            changes.append("added RFC published event at %s" % e.time.strftime("%Y-%m-%d"))
+            rfc_changes.append(
+                f"added RFC published event at {e.time.strftime('%Y-%m-%d')}"
+            )
             rfc_published = True
 
-        for t in ("draft-iesg", "draft-stream-iab", "draft-stream-irtf", "draft-stream-ise"):
-            prev_state = doc.get_state(t)
-            if prev_state is not None:
-                if prev_state.slug not in ("pub", "idexists"):
-                    new_state = State.objects.select_related("type").get(used=True, type=t, slug="pub")
-                    doc.set_state(new_state)
-                    changes.append("changed %s to %s" % (new_state.type.label, new_state))
-                    e = update_action_holders(doc, prev_state, new_state)
-                    if e:
-                        events.append(e)
-            elif t == 'draft-iesg':
-                doc.set_state(State.objects.get(type_id='draft-iesg', slug='idexists'))
-
         def parse_relation_list(l):
             res = []
             for x in l:
-                # This lookup wasn't finding anything but maybe some STD and we know 
+                # This lookup wasn't finding anything but maybe some STD and we know
                 # if the STD had more than one RFC the wrong thing happens
                 #
-                #if x[:3] in ("NIC", "IEN", "STD", "RTR"):
+                # if x[:3] in ("NIC", "IEN", "STD", "RTR"):
                 #    # try translating this to RFCs that we can handle
                 #    # sensibly; otherwise we'll have to ignore them
                 #   l = DocAlias.objects.filter(name__startswith="rfc", docs__docalias__name=x.lower())
-                #else:
+                # else:
                 l = Document.objects.filter(name=x.lower())
```

Comment:
I wonder if we should be filtering this to doc type rfc?

Reply:
I'd say so